packages/ui-docs-client/src/Header/index.js

```diff
@@ -38,11 +43,90 @@ import theme from './theme'
 class Header extends Component {
   static propTypes = {
     name: PropTypes.string.isRequired,
-    version: PropTypes.string
+    version: PropTypes.string,
+    versionsData: PropTypes.shape({
+      latestVersion: PropTypes.string.isRequired,
+      previousVersions: PropTypes.arrayOf(PropTypes.string).isRequired
+    })
   }
 
   static defaultProps = {
-    version: undefined
+    version: undefined,
+    versionsData: {
+      latestVersion: '',
+      previousVersions: []
+    }
+  }
+
+  handleSelect = (_e, [ selectedVersion ]) => {
+    const { versionsData } = this.props
+    const { latestVersion } = versionsData
+    const isSelectedLatestVersion = selectedVersion === latestVersion
+    const splittedUrl = window.location.pathname.split('/').filter(Boolean)
+    const [ versionInPath ] = splittedUrl
+    const isOnLatestVersion = splittedUrl.length === 0
+
+    // 1: instructure.design, latest: v8, selected: v8 -> navigate to instructure.design/#index
```

Comment:
Should we hardcode the latest version number in a comment? This could get outdated really soon

Reply:
Those are just examples of the functionality